### ISSUE TITLE
/tag

### ISSUE THREAD

**Issue by rovertdude** [TotalFreedom/TotalFreedomMod#1588]
I think SA should be blocked from /tag from non-superadmins as well as STA and SRA as we use those now instead of Super Admin or Telnet Admin or Senior Admin. Mostly just helps against impostor tags.

**Comment by superhaystack** [comment]
Good point. Why hasn't it been already blocked?
Vouch.

**Comment by JeromSar** [comment]
No thanks. Tag was provided as an alternative to /nick. We might as well ban /nick following this logic.

Thanks for your suggestion however! :)

**Comment by DUFCLiam** [comment]
@JeromSar, He means block the new tags AKA: SA, SRA, TCA,, etc

**Comment by JeromSar** [comment]
In that case, sure, why not. :)

**Comment by DUFCLiam** [comment]
@JeromSar, But that is a one line thing to block; We cannot do that die to contribution guidelines.

**Comment by JeromSar** [comment]
@DUFCLiam Which means that I'll have to implement it myself.

**Comment by DUFCLiam** [comment]
@JeromSar, Yes, It does mean that.

**Comment by rovertdude** [comment]
I don't believe it would be 1 line, as it would be adding multiple things since it's adding SA, STA, SRA, etc. to it.

**Comment by DUFCLiam** [comment]
@rovertdude It is all in one String and they scan that for it.

**Comment by AvalancheYT** [comment]
If you look at this code -https://github.com/TotalFreedom/TotalFreedomMod/blob/tfm50/src/main/java/me/totalfreedom/totalfreedommod/command/Command_tag.java#L21 - It is a one line code.

**Comment by rovertdude** [comment]
@AvalancheYT oh - my bad then.
